packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -371,7 +372,13 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     clearTimeout(closeTimer.current);
     clearTimeout(touchTimer.current);
     event.persist();
+
+    const prevUserSelect = document.body.style.WebkitUserSelect;
+    // Prevent iOS text selection on long-tap.
+    document.body.style.WebkitUserSelect = 'none';
+
     touchTimer.current = setTimeout(() => {
```

Comment:
If this timer is cancelled, the style is never restored. The initial ref approach seemed fine. We should still restore the value when we cancel the timer and null the "value-to-restore" so that we don't accidentally restore stale values.

Reply:
you are right, I could reproduce the issue. The text selection was disabled on the page after an interrupted interaction.